sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DataSourceV2Strategy.scala

```diff
@@ -588,6 +590,14 @@ class DataSourceV2Strategy(session: SparkSession) extends Strategy with Predicat
 
     case _ => Nil
   }
+
+  private def getCommand(relation: DataSourceV2Relation): Option[RowLevelOperation.Command] = {
```

Comment:
I dont want to walk the graph if its not a merge, even in V2Write requests it.  So hence I need the write node to know what operation it is.